eng/pipelines/windows.yml

```diff
@@ -63,6 +63,13 @@ jobs:
               _helixQueues: $(netcoreappWindowsQueues)
               _publishTests: true
 
+            arm64_Release:
+              _BuildConfig: Release
+              _architecture: arm64
+              _framework: netcoreapp
+              _helixQueues: $(windowsArmQueue)
+              _publishTests: true
```

Comment:
Not sure whether to include this line or not.

Reply:
This is to publish tests assets to azure blob storage for coreclr to run corefx tests in their CI. I don't think it is necessary now, but might be in the future.

Reply:
cc: @mikem8361 do we want to publish assets for arm64 for coreclr testing?